Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Use vLLM's vllm-openai server image as the base
+FROM vllm/vllm-openai:latest
+
+# Set working directory
+WORKDIR /workspace
+
+# Install necessary build dependencies for sentencepiece
+RUN apt-get update && apt-get install -y \
+    pkg-config \
+    cmake \
+    build-essential
+
+# Copy functionary code and requirements into workspace
+COPY . .
+
+# Install additional Python dependencies
+RUN --mount=type=cache,target=/root/.cache/pip \
+    python3 -m pip install -r requirements.txt
```

Comment:
Can you change the dependency installation to follow the instructions in this [branch](https://github.com/MeetKai/functionary/tree/sglang?tab=readme-ov-file#installation)?

Reply:
I will merge that branch before this so you can install following the instructions from [that branch](https://github.com/MeetKai/functionary/tree/sglang?tab=readme-ov-file#installation).

Reply:
That branch has just been merged to main.